core/src/main/java/com/opensymphony/xwork2/interceptor/ValidationAware.java

```diff
@@ -119,7 +119,9 @@ public interface ValidationAware {
      *
      * @return <code>(hasActionErrors() || hasFieldErrors())</code>
      */
-    boolean hasErrors();
+    default boolean hasErrors() {
```

Comment:
Added default implementation which matches the JavaDoc, makes implementing this class simpler